pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -259,7 +259,7 @@ func computeActiveCondition(pa *pav1alpha1.PodAutoscaler, pc podCounts) {
 
 	case pc.ready >= minReady:
 		if pc.want > 0 || !pa.Status.IsInactive() {
-			pa.Status.MarkHasBeenActive()
+			pa.Status.MarkScaleTargetInitialized()
 			// SKS should already be active.
 			pa.Status.MarkActive()
```

Comment:
Could `MarkActive` also set the `MarkScaleTargetInitialized` or do you prefer to keep the two methods separate?

Reply:
I think I prefer them to be separate. I tried that too, but it felt slightly awkward.